src/pile.sol

```diff
@@ -74,7 +74,9 @@ contract Pile is DSNote {
     }
     
     function file(uint fee, uint speed_) public auth note {
-        fees[fee].speed = speed_; 
+        fees[fee].speed = speed_;
+        fees[fee].chi = ONE;
+        drip(fee);
```

Comment:
this drip should be called before changing the speed IMO.

Reply:
Both ways work.